homeassistant/components/fritz/common.py

```diff
@@ -30,16 +30,18 @@
 from homeassistant.helpers import update_coordinator
 from homeassistant.helpers.device_registry import (
     CONNECTION_NETWORK_MAC,
-    async_entries_for_config_entry,
-    async_get,
+    async_entries_for_config_entry as async_entries_for_config_device_entry,
+    async_get as async_get_device_registry,
     format_mac,
 )
 from homeassistant.helpers.dispatcher import dispatcher_send
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_registry import (
     EntityRegistry,
     RegistryEntry,
+    async_entries_for_config_entry as async_entries_for_config_entity_entry,
     async_entries_for_device,
+    async_get as async_get_entity_registry,
 )
```

Comment:
```python
from homeassistant.helpers import entity_registry as er
```
than just use `er.async_get()` and `er.async_entries_for_config_entry()` ...

Reply:
Done